index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="css/style.css">
```

Comment:
- [ ] [OPTIONAL]  Great job, I recommend putting the `<link>` tag after the `<meta>`  because it makes your project more readable and accessible.